README.md

```diff
@@ -2,9 +2,34 @@
 
 A Javascript client library for [Ably Realtime](https://www.ably.io), a realtime data delivery platform.
 
+## Supported platforms
+ 
+ This SDK is compatible with the following platforms:
+ 
+| Node.js  | Browsers           | 
+|----------|--------------------|
+|   4      | IE 8+              |
+|   6      | Chrome 64+         |
+|   8      | Firefox 58+        |
+|   9      | Safari macOS 11    |
+|          | Safari iOS 11      |
+|          | Chrome on Android 6|
+ 
+We maintain compatibility and explicitly support these platform versions, including performing CI testing on all library revisions.
+
+We do not explicitly maintain compatibility with older platform versions; we no longer perform CI testing on iOS7 as of version 1.0.13 (released on February 2nd 2018). Any known incompatibilities with older versions can be found [here](https://github.com/ably/ably-js/issues?q=is%3Aissue+is%3Aopen+label%3A%22compatibility%22).
+
+**Note**: the ably-js browser library contains fallback mechanisms in order to be able to support older browsers; specifically it supports comet-based connections for browsers that do not support websockets, and this includes JSONP for browsers that do not support cross-origin XHR.
+Each of these fallback transport mechanisms is supported and tested on all of the listed target browsers; even when those browsers do not themselves require those fallbacks.
+These mean that the library should be compatible with nearly any recent browser, on the majority of platforms.
```

Comment:
> compatible with nearly any recent browser

Do we not mean nearly every browser being used?  Recent makes it sound like we're only supporting ones released in the last few months / year at best

Reply:
Yes, just remove "recent"

Reply:
> just remove "recent"

We still need to do this